### PR TITLE
More animal skulls

### PR DESCRIPTION
**Pull request by Karol1223**
#### Summary
Content "Adds more animal skulls"

#### Purpose of change
#61237 added wolf skulls and it really made me think. It also made me sad we didn't have cow skulls. What is a post-apo game without cow skulls? So anyway I fixed that.

#### Describe the solution
Adds several new skulls, these include:
- Rodent (rat, squirrel and chipmunk)
- Small Feline (housecat, I think bobcat is slightly too big to fall in the same category but may change it)
- Small Canine (puppies, small dogs and foxes)
- Tiny Canine (Chihuahua, small dog breed puppies)
- Bovine (just cows, really, but it sounds more professional)
- Horse
- Goat
- Sheep
- Pig (both pigs and wild boars)
- Deer (deer and reindeer as well as dermatik incubator deer, I may add these to spideer? but not sure)
- Moose
- Tusked Moose

ALSO
- Renamed the wolf skull to large canine skull and increased the volume to 4l
- Gave most of the dogs + dermatik incubator dog large canine skulls
- Added some (not all, just the ones I'd expect in the wild and the cow) new skulls to the mammal remains itemgroup
- Increased the volume of the wolf skull mask or whatever its called to 4l from 3l so that it doesn't stand out being less volume than the skull used to craft it (it is less volume than the skull used to craft it since its missing the lower jaw but adding the parts needed to make it a headwear probably about equals out)

#### Describe alternatives you've considered
More skulls. Bear, Large Feline for cougars and the such, Medium Feline for Bobcat, Mink, Beaver etc etc
And like stated before Deer skull for Spideer but I am not sure how they look skeletally so I didn't want to touch that just yet

#### Testing
The skulls appear just fine, disassemble just fine, overall looks good to me

### CODE REVIEW

**Comment by TheShadowFerret** [comment]
Do small mutants get their own skull helmet 👉👈

**Comment by Karol1223** [comment]
> Do small mutants get their own skull helmet 👉👈

Apparel scares me, honestly, I'm here just for the skulls themselves. Though using a chihuahua skull instead of a wolf skull for mouse mutants does seem hilarious and you got me thinking...

File: data/json/items/comestibles/carnivore.json
Comment (chaohedgehogs) [inline data/json/items/comestibles/carnivore.json:1818]:
```suggestion
    "name": { "str": "rodent skull" },
```

File: data/json/items/comestibles/carnivore.json
Comment (chaohedgehogs) [inline data/json/items/comestibles/carnivore.json:1819]:
```suggestion
    "description": "The skull of a small rodent, most likely a rat.",
```

**Comment by Karol1223** [comment]
Converting to draft so that I can work on canines a bit more. Will rename the small canine skulls to medium canine skulls and add new small ones for chihuahuas and dachshund/beagle puppies. Also give coyotes the medium canine skulls because large ones are a bit large for them.

**Comment by Karol1223** [comment]
> Wow, a moose skull is the same weight as 65 generic bones? If those numbers are right I think this is good to go

The numbers are from google, unfortunately, I have no access to a moose skull to check for myself, and estimates from 'experts' online vary between 20 and 70 pounds for antlers _alone_. In retrospect, I may have made it too light if anything but I think for a generic moose that's not explicitly hunted for its antlers the number given is a reasonably good assumption.

**Review by LyleSY — APPROVED** [review]

**Comment by Fris0uman** [comment]
Skulls shoud probably go in their own json file instead of carnivore.json, there's enough of them and they're not really food

**Comment by Karol1223** [comment]
> Skulls shoud probably go in their own json file instead of carnivore.json, there's enough of them and they're not really food

I absolutely agree, actually, I was thinking about that. Should I just move the new ones to their own seperate file or should I move the human and existing canine skulls there as well? I suspect the first but I'd rather make sure

**Comment by MNG-cataclysm** [comment]
> I absolutely agree, actually, I was thinking about that. Should I just move the new ones to their own separate file or should I move the human and existing canine skulls there as well? I suspect the first but I'd rather make sure

I'd move all the skulls into the file; for ease of future organization.

**Comment by Karol1223** [comment]
> I'd move all the skulls into the file; for ease of future organization.

I'd like Fris' or someone else's with merge rights opinion on this. I know for a fact moving file contents fucks with git blame and can cause issues for others so I'd rather not do it without their approval.

**Comment by Karol1223** [comment]
@Fris0uman you asked to be pinged here when I move the skulls over to a new file, its done.

**Pull request merged**
